l1-contracts/src/UniFiAVSManager.sol

```diff
@@ -439,26 +465,40 @@ contract UniFiAVSManager is UniFiAVSManagerStorage, IUniFiAVSManager, UUPSUpgrad
         returns (address[] memory restakedStrategies)
     {
         OperatorDataExtended memory operatorData = _getOperator(operator);
-        IStrategy[] memory strategies = new IStrategy[](1);
-        strategies[0] = IStrategy(BEACON_CHAIN_STRATEGY);
-        uint256[] memory shares = EIGEN_DELEGATION_MANAGER.getOperatorShares(operator, strategies);
+        UniFiAVSStorage storage $ = _getUniFiAVSManagerStorage();
+        if (operatorData.isRegistered) {
```

Comment:
Instead of fetching operatorData on 467. I would have done
```
        if (
            AVS_DIRECTORY.avsOperatorStatus(address(this), operator)
                != IAVSDirectory.OperatorAVSRegistrationStatus.REGISTERED
        ) {
            return new address[](0);
        }
        ```